### ISSUE TITLE
Add Flores to operator definitions

### ISSUE THREAD

**Issue by ElBeenMachine** [billy-yoyo/RainbowSixSiege-Python-API#80]
I have the r6sapi downloaded locally and I was wondering where you get the data for the operator definitions so I could add my own when the operators come out? This would be some really useful information. Thanks in advance

**Comment by Jackywathy** [comment]
I don't believe there is a official list of operator definitions: we've had to guess them now that the old ubisoft stat page got replaced by the new one. It would probably be something like flores... and the guns should be the same

**Comment by ElBeenMachine** [comment]
What about the IDs and stuff like that?
